src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -1470,6 +1470,9 @@ void scrollElementToFirstCellAndSendKeys(WebElement spreadsheetElement, String v
         void registerHandlers() {
             checkState(!hasHandlers(), "`ajaxStart` and `ajaxStop` handlers need only be added once to the document.");
 
+            // wait for all AJAX requests to complete
+            waitFor(driver -> (Boolean) ((JavascriptExecutor) driver).executeScript("return $.active === 0"));
```

Comment:
Let's extract this as `waitForAllRequestsComplete`?
And maybe we should explain `$.active` a little since it is not an actual API that JQuery expose? 
```Javascript
            // wait for all AJAX requests to complete.`$.active` is a counter for holding the number of active AJAX requests
            // but is not documented because it is used by JQuery internally.
```